packages/dialog/test/lion-dialog.test.js

```diff
@@ -0,0 +1,79 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import '../lion-dialog.js';
+
+describe('lion-dialog', () => {
+  describe('Basic', () => {
+    it('should not be shown by default', async () => {
+      const el = await fixture(html`
+        <lion-dialog>
+          <div slot="content" class="dialog">Hey there</div>
+          <lion-button slot="invoker">Popup button</lion-button>
+        </lion-dialog>
+      `);
+      expect(el._overlayCtrl.isShown).to.be.false;
+    });
+
+    it('should show content on invoker click', async () => {
+      const el = await fixture(html`
+        <lion-dialog>
+          <div slot="content" class="dialog">
+            Hey there
+          </div>
+          <lion-button slot="invoker">Popup button</lion-button>
+        </lion-dialog>
+      `);
+      const invoker = el.querySelector('[slot="invoker"]');
+      invoker.click();
+      await el.updateComplete;
```

Comment:
Not as a consequence of this MR but as a consequence of the OverlayController now using nodes instead of templates. But I'll remove the redundant awaits :). Because we want them to fail if we ever go to something that happens async again.

Reply:
ah to clarify: these tests are just smoke tests copied from OverlayController